pkg/virt-controller/watch/migration_test.go

```diff
@@ -2162,6 +2161,59 @@ var _ = Describe("Migration watcher", func() {
 			expectTargetPodWithSELinuxLevel(vmi.Namespace, vmi.UID, migration.UID, "")
 		})
 	})
+
+	Context("Secondary queue", func() {
```

Comment:
Nice!
I wonder if it'd be valuable to create another test with multiple pending migrations that are expected to be flushed, just to cover this case as well (or maybe I'm being over-paranoid here?)